data/card_catalogue/upgradeable-1.json

```diff
@@ -599,42 +364,12 @@
       "image": "assets/core/card/illustration/Endless blows.png"
     }
   },
-  {
-    "title": "Four Leaf Clover",
-    "upgrade": "Rabbit's Foot",
-    "tier": "blue",
-    "details": {
-      "rules": "Place 3 green cards on the table. You may cast one of them.",
-      "flavour": "No more of this bush league nonsense. We need some real power.",
-      "image": "assets/core/card/illustration/Four Leaf Clover.png"
-    }
-  },
-  {
-    "title": "Rabbit's Foot",
-    "upgrade": "Jackpot",
-    "tier": "green",
-    "details": {
-      "rules": "Place 3 Red cards on the table. You may cast one of them.",
-      "flavour": "I might be able to pull it off- but just this once.",
-      "image": "assets/core/card/illustration/Rabbit_s Foot.png"
-    }
-  },
-  {
-    "title": "Jackpot",
-    "upgrade": "",
-    "tier": "red",
-    "details": {
-      "rules": "Place 3 Gold cards on the table. You may cast one of them.",
-      "flavour": "Do you believe in miracles?",
-      "image": "assets/core/card/illustration/Jackpot.png"
-    }
-  },
   {
     "title": "Gust of Wind",
     "upgrade": "Gale",
     "tier": "blue",
     "details": {
-      "rules": "Knock target visible character airborne. If this character is already airborne, knock them back 5 squares",
+      "rules": "Throw target visible character up into the air",
```

Comment:
Simplified the wording on this set. Moved the knockback into the green passive, so it only kicks it when you're actually building a wind wizard.